featuretools/primitives/standard/transform_primitive.py

```diff
@@ -181,7 +319,14 @@ def get_function(self):
 
 
 class NumWords(TransformPrimitive):
-    """Returns the number of words in a given string by counting the spaces.
+    """Determines the number of words in a string by counting the spaces.
```

Comment:
maybe add a line and example with some punctuation so it's clear how that is handled